config.py

```diff
@@ -592,6 +592,9 @@ def create_transaction(db):
     RECAPTCHA_SITE_KEY: Optional[str] = None
     RECAPTCHA_SECRET_KEY: Optional[str] = None
 
+    # List of users allowed to pass through recaptcha security check to enable org/user creation via API
+    RECAPTCHA_WHITELIST_USERS: List[str] = []
```

Comment:
RECAPTCHA_WHITELISTED_NAMESPACES

Reply:
should it be `NAMESPACES` instead of `USERS` ?

Reply:
wouldn't `Namespaces` include orgs? However this list would only take in `users` as input?

Reply:
Right, RECAPTCHA_WHITELISTED_USERS should be fine then.